pkgs/development/python-modules/matplotx/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, matplotlib
+, numpy
+, networkx
+, pypng
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "matplotx";
+  version = "0.3.10";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "nschloe";
+    repo = "matplotx";
+    rev = "v${version}";
+    hash = "sha256-EWEiEY23uFwd/vgWVLCH/buUmgRqz1rqqlJEdXINYMg=";
+  };
+
+  propagatedBuildInputs = [
+    matplotlib
+    numpy
+    networkx
+    pypng
+    scipy
+  ];
+
+  pythonImportsCheck = [ "matplotx" ];
+
+  meta = {
+    homepage = "https://github.com/nschloe/matplotx";
+    description = "More styles and useful extensions for Matplotlib";
```

```suggestion
    description = "More styles and useful extensions for Matplotlib";
    changelog = "https://github.com/nschloe/matplotx/releases/tag/v${version}"
```
It is better to add a changelog to the meta.